server/memphis_handlers_stations.go

```diff
@@ -273,6 +281,16 @@ func (s *Server) createStationDirectIntern(c *client,
 		replicas = 1
 	}
 
+	if csr.RetentionType == "message_age_sec" {
+		err = validateIdempotencyWindow(csr.RetentionValue, csr.IdempotencyWindow)
```

Comment:
insert the retention type check into the validation function

Reply:
I was thinking about doing it, but in which condition I call validateIdempotencyWindow?

Reply:
call it anyway